app/models/liked_art.rb

```diff
@@ -0,0 +1,4 @@
+class LikedArt < ApplicationRecord
```

Comment:
Possibly rename? We would save any rated art no matter if it was liked or not, correct? Possible option... RatedArt

Reply:
Or maybe not?... let's chat we have a question that may make our earlier comment obsolete.